docker-compose.yml

```diff
@@ -21,12 +21,13 @@ services:
     build:
       context: ./
       dockerfile: ./node/Dockerfile
+    command: bash -c "rm -rf /home/node/app/node_modules/* && npm run start"
```

Comment:
Why is the rm needed if we don't use an anonymous volume?

Reply:
just to be sure, the /home/node/app/node_modules should be empty but you never know.

"make sure the volume stays clean by deleting its contents on every start (keeping the local ones but deleting it within the container)"
See the last text. 
https://burnedikt.com/dockerized-node-development-and-mounting-node-volumes/#improving-the-workaround

Reply:
this is not removing the container node_modules, that are in another path Jon. Just removing any file that could be left from the user.

Reply:
Here it is more explained:
Also, keep in mind that we still require the exclude volume to be mounted (and be empty at all times) as Node.js’s module resolver would prioritize any module within the exclude volume over the parent directory. That means if someone was to (accidentally) run an npm install within the running container at the /usr/local/app folder level, we would again pollute the exclude folder and introduce the third point of failure once again. This is why it might make sense to always clear out the contents of the exclude volume on every container start to ensure a clean state.